operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ Entries should include a reference to the Pull Request that introduced the chang
   Mimir under the `metamonitoring` tenant; this enhancement does not apply to GEM. #3176
 * [ENHANCEMENT] Improve default rollout strategies. Now distributor, overrides_exporter, querier, query_frontend, admin_api, gateway, and graphite components can be upgraded more quickly and also can be rolled out with a single replica without downtime. #3029
 * [ENHANCEMENT] Metamonitoring: make scrape interval configurable. #2945
+* [ENHANCEMENT] Add ability to manage PrometheusRule for metamonitoring with Prometheus operator from the Helm chart. If `serviceMonitor` is enabled, rules standard to Mimir will be deployed. The accompanying alerts are disabled by default but can be created with `serviceMonitor.mimirAlerts` set to `true`. To disable the default rules, set `mimirRules` to `false`. #2134 #2609
```

Comment:
Update according to https://github.com/grafana/mimir/pull/2609#discussion_r1006611988 plz